src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -530,9 +551,20 @@ parse_param(Key, Val, Args, IsDecoded) ->
             JsonKey = ?JSON_DECODE(Val),
             Args#mrargs{start_key = JsonKey, end_key = JsonKey};
         "keys" when IsDecoded ->
-            Args#mrargs{keys = Val};
+            case Val of
+                [Val1] ->
```

Comment:
`Key` would be a better name for `Val1` here.

Reply:
Thanks for your comments, I tried to rewrite them here https://github.com/apache/couchdb/pull/4494
Change `[Val1]` to `[SingleKey]` because `Key` is already used.